packages/server/create-request/src/node.ts

```diff
@@ -127,3 +128,39 @@ export const createRequest: RequestCreator<typeof nodeFetch> = (
 
   return sender;
 };
+
+export const createUploader: RequestUploader = (path: string) => {
```

Comment:
What are the differences between implementations in web and node? Can it be unified?